wireguard-manager.sh

```diff
@@ -591,8 +591,8 @@ if [ ! -f "${WIREGUARD_CONFIG}" ]; then
     if [ -f "${WIREGUARD_INTERFACE}" ]; then
       echo "Do you want to disable IPv4 or IPv6 on the server?"
       echo "  1) No (Recommended)"
-      echo "  2) Disable IPV4"
-      echo "  3) Disable IPV6"
+      echo "  2) Disable IPv4"
```

Comment:
We are disabling ipv4 and only enabling ipv6 

you just forwarded ipv4 when it says to disable ipv4